lib/sorcery/controller/submodules/jwt_auth.rb

```diff
@@ -0,0 +1,85 @@
+module Sorcery
+  module Controller
+    module Submodules
+      module JwtAuth
+        def self.included(base)
+          base.send(:include, InstanceMethods)
+        end
+
+        module InstanceMethods
+          # This method return generated token if user can be authenticated
+          def jwt_auth(*credentials)
```

Comment:
I think slightly more appropriate name for the method would be `jwt_authenticate`. Simply to stay consistent with `User.authenticate`.

Reply:
Method should also allow users to get login failure reason, similar to what `login` method does.